### ISSUE TITLE
Keyboard navigation doesnt work

### ISSUE THREAD

**Issue by SakalCZ** [jackmoore/colorbox#416]
I don't know if have too heavy Javascript around my website, but keyboard navigation doesn't work for me. Everything else works fine.

I fixed this problem by changing section "Key Bindings" (line 474):
$prev.click();  ----> to $prev.trigger('click');
$next.click();  ----> to $next.trigger('click');

btw i'm glad you are developing this project, wish you the best ;-)
